app/assets/javascripts/darkswarm/directives/country_selector.js.coffee

```diff
@@ -0,0 +1,11 @@
+Darkswarm.directive "countrySelector", () ->
```

Comment:
Maybe we can save some code here if we set the country at the beginning in https://github.com/openfoodfoundation/openfoodnetwork/blob/4b8146dd00239a86b3461fb5c1b203f715abb3f7/app/assets/javascripts/darkswarm/services/enterprise_registration_service.js.coffee#L8. Would that work?